### ISSUE TITLE
Feedback Assistant Support

### ISSUE THREAD

**Issue by DaveWoodCom** [timburks/openradar#43]
Since Apple has removed the old bug reporter, and replaced it with the new Feedback Assistant https://feedbackassistant.apple.com/ there are a few changes required.

The main issue is that there's a new numbering system eg: FB0000000. Each bug report now begins with FB (to differentiate it from the old radar system). When submitting reports via openradar, we get an error if we include the FB prefix. If we omit the FB we can file the report, but then there's no way to know if it's a new Feedback Assistant report, or an old radar report.

**Comment by timburks** [comment]
I'm guessing that the numbering started over. Is that correct?

**Comment by DaveWoodCom** [comment]
Yes, the numbers have apparently reset, though are quite high since internally they seem to have mapped existing (current/not deleted) radars to new feedback reports.

The other main difference is the prefix of FB on new reports (which lets them [and us] know which system the report is in). rdar:// no long appears to work for them internally either, though it would be good to get someone inside Apple to confirm (and maybe let us know if they have a new prefix, fb:// ?)

**Comment by manicmaniac** [comment]
How about just migrating the current post numbers from `integer` to `string` and permit inputting `string` as a post number?
In that way we can keep older posts and accept the new format.

**Comment by manicmaniac** [comment]
The URL scheme of Feedback Assistant is `applefeedback://`.
I confirmed that this URL scheme opens Feedback Assistant but I don't know how to jump to the specific number.

https://developer.apple.com/bug-reporting/
>  it can be launched with Spotlight or the URL scheme applefeedback://.

**Comment by sethfri** [comment]
Of note, Feedback Assistant doesn't ask for the `Product Version` and `Reproducible` fields anymore when I've used it, so it may be worth removing those from OpenRadar

**Comment by timburks** [comment]
@manicmaniac Thanks for the PR! I've merged and deployed it, please ping me if you notice any problems. (I unfortunately miss a lot of github email, so the best way to reach me is at https://twitter.com/timburks). I'll leave this open until someone comments that this is working.
